src/components/summary-list/index.md

```diff
@@ -8,9 +8,36 @@ layout: layout-pane.njk
 ---
 
 {% from "_example.njk" import example %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
 
 Use a summary list to summarise information, for example, a user’s responses at the end of a form.
 
+{% set wcagCallout %}
+  <p class="govuk-!-margin-bottom-2">
+    {{ govukTag({
+      text: "WCAG 2.2"
+    })}}
+  </p>
+
+  ### Accessibility considerations for row actions
+
+  There's a few things to keep in mind to ensure that users can successfully:
+
+  - [interact with row action](/components/summary-list/#:~:text=Keep%20card%20and%20row%20actions%20as%20%E2%80%98inline%20links%E2%80%99.%20This%20is%20to%20make%20sure%20that%20sets%20of%20actions%20reflow%20properly%20on%20different%20screen%20sizes.%20This%20relates%20to%20WCAG%202.2%20success%20criterion%202.5.8%20Target%20Size%20(minimum).)
+  - [change information they've previously given in an answer](/components/summary-list/#:~:text=Do%20not%20pre%2Dpopulate%20if%20the%20information%20is%20no%20longer%20valid%2C%20or%20when%20pre%2Dpopulating%20would%20be%20a%20major%20safety%20or%20security%20concern.%20This%20is%20to%20comply%20with%20WCAG%202.2%20success%20criterion%203.3.7%20Redundant%20Entry.)
```

Comment:
Noticed that these two links use text fragments to link to other parts of the current page.

Text fragments are not currently supported in Firefox (and will never be supported in IE 11, which we still _technically_ support). In these browsers, the user will be jumped back to the top of the page instead.

These should probably be swapped out for hardcoded IDs/anchors to ensure support for all of our browsers. This wouldn't do the text highlighting effect that fragments do, but this can be mimicked using `:target` CSS styles. 

Using IDs would also mean that the fragment wouldn't need to be kept in sync with any changes to the actual text, making them a little easier to maintain.

Reply:
@querkmachine is there a way we can have a fall-back option for browsers that don't support the text fragments? @romaricpascal suggested it might be possible to use some javascript to mimic this behaviour.

> On the bright side, the Chrome folks also provide some [JavaScript that implements the feature for browsers that do not support it](https://github.com/GoogleChromeLabs/text-fragments-polyfill). Depending on the weight, that could make an easy win.